sdks/unity/AgonesSdkClient.cs

```diff
@@ -0,0 +1,179 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Newtonsoft.Json;

```

Comment:
I have several times found myself importing Newtonsoft into a Unity program, only to realize that Unity's built-in `JsonUtility.ToJson` and `JsonUtility.FromJson<T>` do provide the functionality I need. This obviates the need to install a separate NuGet package, which is not very well documented/supported in Unity, I find.

From what I can see `JsonConvert` being used for here, it looks like `JsonUtility` would cover this, but let me know if it doesn't work for some reason.